### ISSUE TITLE
Should this be archived / deleted?

### ISSUE THREAD

**Issue by gmarkall** [numba/numba-webpage#10]
It presently seems to serve an outdated page with broken image links at http://numba.pydata.org/numba-webpage/

I think the current page is served from https://github.com/numba/numba.github.com

**Comment by seibert** [comment]
Yes, I'm not sure why there are two of these.  I will archive this repo.
